test/core/utils/get-base-lang.js

```diff
@@ -0,0 +1,18 @@
+describe('axe.utils.getBaseLang', function() {
```

Comment:
Also check things with multiple dashes. What should this do 'foo-bar-baz'? It's clearly invalid, so maybe this needs to return empty or null?